TrainingApp/src/store/reducers/orders.js

```diff
@@ -0,0 +1,28 @@
+import { ADD_ORDER, SET_ORDERS } from '../actions/orders';
+import Order from '../../models/order';
+
+const initialState = {
+  orders: []
+};
```

Comment:
better to have following props in initial state 
ex:
  isFetching: false,
  orders: [],
  ordersSuccess: false,
  ordersError: null

Reply:
Made necessary changes